ui/apps/platform/src/Containers/NetworkGraph/deployment/ExternalFlows.tsx

```diff
@@ -71,18 +196,43 @@ function ExternalFlows({ deploymentId, timeWindow }: ExternalFlowsProps) {
                                 flowCount={totalAnomalous}
                                 emptyStateMessage="No anomalous flows."
                                 tableState={anomalous.tableState}
+                                selectedPageAll={areAllPageAnomalousSelected}
```

Comment:
Is there a generic prop name that has `are` as prefix? Like `areAllSelected` but you might do better.

Reply:
I updated this in my other PR so that I could get this PR merged